frontend/micro-ui/web/micro-ui-internals/packages/modules/AttendenceMgmt/src/config/inboxConfig.js

```diff
@@ -1,177 +1,232 @@
-                            label: "ATM_IMPLEMENTING_AGENCY",
-                            type: "dropdown",
-                            isMandatory: false,
-                            disable: false,
-                            populators: {
-                              name: "iaip",
-                              optionsKey: "name",
-                              optionsCustomStyle : {
-                                top : "2.3rem"
-                              },
-                              mdmsConfig: {
-                                masterName: "Department",
-                                moduleName: "common-masters",
-                                localePrefix: "COMMON_MASTERS_DEPARTMENT",
-                              }
+                        "fields" : [
+                            {
+                                "label": "ATM_MUSTER_ROLL_ID",
+                                "type": "text",
+                                "isMandatory": false,
+                                "disable": false,
+                                "preProcess": {
+                                    "convertStringToRegEx": [
+                                        "populators.validation.pattern"
+                                    ]
+                                },
+                                "populators": {
+                                    "name": "musterRollNumber",
+                                    "error": "PROJECT_PATTERN_ERR_MSG",
+                                    "validation": {
+                                        "pattern": "MR\\/[0-9]+-[0-9]+\\/[0-9]+\\/[0-9]+",
+                                        "minlength": 2
+                                    }
+                                }
+                            },
+                            {
+                                "label": "ATM_REGISTER_NAME",
+                                "type": "text",
+                                "isMandatory": false,
+                                "disable": false,
+                                // "preProcess": {
+                                //     "convertStringToRegEx": [
+                                //         "populators.validation.pattern"
+                                //     ]
+                                // },
+                                "populators": {
+                                    "name": "attendanceRegisterName",
+                                    "error": "PROJECT_PATTERN_ERR_MSG",
+                                    // "validation": {
+                                    //     "pattern": "PR\\/[0-9]+-[0-9]+\\/[0-9]+\\/[0-9]+",
+                                    //     "minlength": 2
+                                    // }
+                                }
+                            },
+                            //here need to render a dropdown for org 
+                            {
+                            "label": "COMMON_ORG_NAME",
```

Comment:
@hariomsinha-eGov @prachirane-eGov Hi guys I have added a new dropdown in renderformfields called apidropdown in case we need to show data from any api other than mdms.